components/controls/Modal.css

```diff
@@ -88,11 +88,6 @@
   margin-right: var(--sbw);
 }
 
-.modal-head {
-  padding: calc(4 * var(--gridSize));
-  padding-bottom: 0;
-}
-
```

Comment:
This is a breaking change. It should not be removed, **unless** all modals are updated before updating sonar-ui-common. So, if this makes it into SUC, we (SC and SQ) would better make sure all our modals are ready for it!